src/Storefront/Framework/Twig/Extension/UrlEncodingTwigFilter.php

```diff
@@ -65,6 +66,9 @@ public function encodeMediaUrl(?MediaEntity $media): ?string
         if ($media === null || !$media->hasFile()) {
             return null;
         }
+        if (Feature::isActive('v6.8.0.0')) {
+            return $media->getUrl();
+        }
 
         return $this->encodeUrl($media->getUrl());
```

Comment:
Small nitpick: its better to invert this if so for the 6.8 major deprecation removal we can just remove the if